Kitodo/src/main/java/de/sub/goobi/forms/BenutzergruppenForm.java

```diff
@@ -136,6 +139,41 @@ public int getUserGroupId() {
         return this.userGroupId;
     }
 
+    /**
+     * Return the list of available authorization levels and the list of
+     * authorization levels currently assigned to 'myBenutzergruppe' as a combined
+     * 'DualListModel' that is used by the frontend for authorization management of
+     * user groups utilizing a PrimeFaces PickList object.
+     *
+     * @return DualListModel of available and assigned authorization levels
+     */
+    public DualListModel<Authorization> getAuthorizations() {
+        List<Authorization> assignedAuthorizations = this.myBenutzergruppe.getAuthorizations();
+        List<Authorization> availableAuthorizations = serviceManager.getAuthorizationService().getAll();
+        availableAuthorizations.removeAll(assignedAuthorizations);
+        return new DualListModel<>(availableAuthorizations, assignedAuthorizations);
+    }
+
+    /**
+     * Assign the target property of given DualListModel of authorizations to
+     * 'myBenutzergruppe' using a PrimeFaces PickList object.
+     *
+     * @param authorizations
+     *            list of authorizations assigned to 'myBenutzergruppe'
+     */
+    public void setAuthorizations(DualListModel<Authorization> authorizations) {
+        try {
+            List<Authorization> assignedAuthorizations = new ArrayList<>();
```

Comment:
This method probably can be simplified to sometgin like that:
`public void setAuthorizations(DualListModel<Authorization> authorizations) {
        List<Authorization> assignedAuthorizations = new ArrayList<>();
        for (Authorization object : authorizations.getTarget()) {
            assignedAuthorizations.add(object); 
        }
        this.myBenutzergruppe.setAuthorizations(assignedAuthorizations);
    }`

Reply:
No, unfortunately, I think that change is not possible. `getTarget` returns just the values of the list items in the PickLists "target" list. These values are just the id's of the authorization objects, not the whole authorization objects themselves.